htsget-http-actix/src/get.rs

```diff
@@ -0,0 +1,49 @@
+use actix_web::{
+  http::StatusCode,
+  web::{Data, Json, Path, Query},
+  Responder,
+};
+use htsget_http_core::{get_response_for_get_request, Endpoint};
+use htsget_search::htsget::HtsGet;
+use std::collections::HashMap;
+
+/// GET request reads endpoint
+pub async fn reads<H: HtsGet>(
+  request: Query<HashMap<String, String>>,
+  Path(id): Path<String>,
+  shared_state: Data<H>,
```

Comment:
Very good job with injecting HtsGet in a generic way ;-) 

I would rename `shared_state` to `htsget`.